src/Symfony/Component/Debug/ExceptionHandler.php

```diff
@@ -36,7 +36,7 @@ class ExceptionHandler
     private $caughtLength;
     private $fileLinkFormat;
 
-    public function __construct(bool $debug = true, string $charset = null, $fileLinkFormat = null)
+    public function __construct($debug = true, $charset = null, $fileLinkFormat = null)
```

Comment:
Should be reverted

Reply:
Eeeh. I asked about this )

Reply:
https://github.com/symfony/symfony/pull/27071#issuecomment-386484021

Reply:
No problem, BTW